src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.cs

```diff
@@ -276,7 +276,13 @@ protected virtual void OnTextChanged(string oldValue, string newValue)
 		{
 			UpdateInlines(newValue);
 
-			Selection = new Range(0, 0);
+#if __SKIA__
```

Comment:
Can this be covered with a test?

Reply:
Is it a visual glitch, or can we test the changes of the `Selection` property? Or something similar? 

Also why does setting the selection this way fix the problem?